conf/cndp_upf_4worker.jsonc

```diff
@@ -0,0 +1,289 @@
+{
+    // (R) - Required entry
+    // (O) - Optional entry
+    // All descriptions are optional and short form is 'desc'
+    // The order of the entries in this file are handled when it is parsed and the
+    // entries can be in any order.
+
+    // (R) Application information
+    //    name        - (O) The name of the application
+    //    description - (O) The description of the application
+    "application": {
+        "name": "cndp-upf",
+        "description": "CNDP OMEC-UPF configuration"
+    },
+
+    // (O) Default values
+    //    bufcnt - (O) UMEM default buffer count in 1K increments
+    //    bufsz  - (O) UMEM buffer size in 1K increments
+    //    rxdesc - (O) Number of RX ring descriptors in 1K increments
+    //    txdesc - (O) Number of TX ring descriptors in 1K increments
+    //    cache  - (O) MBUF Pool cache size in number of entries
+    //    mtype  - (O) Memory type for mmap allocations
+    "defaults": {
+        "bufcnt": 32,
+        "bufsz": 2,
+        "rxdesc": 2,
+        "txdesc": 2,
+        "cache": 256,
+        "mtype": "2MB"
+    },
+
+    // List of all UMEM's to be created
+    // key/val - (R) The 'key' is the name of the umem for later reference.
+    //               The 'val' is the object describing the UMEM buffer.
+    //               Multiple umem regions can be defined.
+    // A UMEM can support multiple lports using the regions array. Each lports can use
+    // one of the regions.
+    //    bufcnt  - (R) The number of buffers in 1K increments in the UMEM space.
+    //    bufsz   - (R) The size in 1K increments of each buffer in the UMEM space.
+    //    mtype   - (O) If missing or empty string or missing means use 4KB or default system pages.
+    //    regions - (O) Array of sizes one per region in 1K increments, total must be <= bufcnt
+    //    rxdesc  - (O) Number of RX descriptors to be allocated in 1K increments,
+    //                  if not present or zero use defaults.rxdesc, normally zero.
+    //    txdesc  - (O) Number of TX descriptors to be allocated in 1K increments,
+    //                  if not present or zero use defaults.txdesc, normally zero.
+    //    description | desc - (O) Description of the umem space.
+        "umems": {
```

Comment:
Can you please make sure that the JSONC files are properly formatted? I was expecting to see this line to start after 4 spaces (same as cndp_upf_1worker.jsonc), but here it starts after 8 spaces. Moreover, "regions" in here sets one value after another "16,16", while in the 1worker file, there is one value line. Additionally, make sure the comments are not too long (single line), if needed split them in multiple lines

Reply:
My comment is primary when comparing `cndp_upf_1worked.jsonc and cndp_upf_4worked.jsonc` formatting. Since both files are basically the same, I was expecting for both have the same formatting, However, as shown in the figure below, they have different formatting for the same fields and one of those fields is `umems`
![image](https://user-images.githubusercontent.com/87488727/225693980-f3aab1c7-b1fb-43bb-8459-3baa788c7a47.png)

Reply:
So specific reason I can think for for different formatting. Both are valid jsonc constructs. eslint didn't give any error  for both